pkg/render/kubecontrollers/kube-controllers.go

```diff
@@ -614,6 +633,9 @@ func (c *kubeControllersComponent) kubeControllersVolumes() []corev1.Volume {
 	if c.cfg.TrustedBundle != nil {
 		volumes = append(volumes, c.cfg.TrustedBundle.Volume())
 	}
+	if c.cfg.MetricsServerTLS != nil {
+		volumes = append(volumes, c.cfg.MetricsServerTLS.Volume())
```

Comment:
It needs to also be added to the `annotations()` func.

Reply:
Done